Windows/MainWindow.xaml.cs

```diff
@@ -95,11 +98,31 @@ public partial class MainWindow {
             }
         }
 
-        public bool SingleClickToEditMode { get; set; } = true;
```

Comment:
Don't change the defaults.

Reply:
I'm only trying to honor your work by contributing my modifications to it.  I understand that you don't have to accept every little change I have modified.